x-pack/docs/en/security/authorization/mapping-roles.asciidoc

```diff
@@ -36,17 +36,14 @@ To use file based role-mappings, you must configure the mappings in a YAML file
 and copy it to each node in the cluster. Tools like Puppet or Chef can help with
 this.
 
-By default, role mappings are stored in `ES_PATH_CONF/x-pack/role_mapping.yml`,
+By default, role mappings are stored in `ES_PATH_CONF/role_mapping.yml`,
```

Comment:
Because of this change, this will need to be manually edited for versions pre 6.3

Reply:
This change was back-ported to 5.6, 6.0, 6.1, and 6.2 in x-pack-elasticsearch via commit 84e0ddc84357d5cb80cef3f57362c5de4572dab6